.github/workflows/lint-and-test.yml

```diff
@@ -77,29 +84,43 @@ jobs:
     - name: Install PHP dependencies
       run: |
         export PATH="$HOME/.composer/vendor/bin:$PATH"
-        composer install --no-interaction
+        composer install --no-dev --no-interaction
         ulimit -n 4096
 
     - name: Run ESLint and Stylelint
       run: npm run lint
       if: matrix.experimental == false
 
     - name: Run PHP CodeSniffer
-      run: composer standards
+      run: |
+        composer update --dev pressbooks/coding-standards
```

Comment:
update to latest version of coding standards -- not sure if there's a better way to do this (i.e. ensure that we're installing the version of coding-standards declared in composer lock for example). maybe `composer install --dev pressbooks/coding-standards`?